src/pylorax/installer.py

```diff
@@ -142,6 +142,17 @@ class QEMUInstall(object):
                  "aarch64": "qemu-system-aarch64",
                  "ppc64le": "qemu-system-ppc64"
                 }
+    COMPATIBLE_ARCHS = {"x86_64": [ "x86_64", "i386" ],
```

Comment:
Small style thing, no need for spaces between `[` and the contents of the list. We've (I!) have been inconsistent about this, but no spaces looks nicer.